compaction/compaction_strategy.cc

```diff
@@ -174,8 +177,11 @@ void compaction_strategy_impl::validate_options_for_strategy_type(const std::map
         case compaction_strategy_type::time_window:
             time_window_compaction_strategy::validate_options(options, unchecked_options);
             break;
-        default:
+        case compaction_strategy_type::incremental:
+            incremental_compaction_strategy::validate_options(options, unchecked_options);
             break;
+        case compaction_strategy_type::null:
+            return;
```

Comment:
`default:` had `break` before the change.
We shouldn't change that in this patch.
And it's nit a bad idea to erase the `NullCompactionStrategy` class form the options, same as for any other strategy, even if it's not strictly requried.